packages/jest/examples/multiple-apps/package.json

```diff
@@ -7,41 +7,39 @@
     "build": "yarn ng build",
     "test": "yarn ng test",
     "lint": "yarn ng lint",
-    "e2e": "yarn ng e2e",
-    "postinstall": "ngcc"
```

Comment:
Why the `ngcc` postinstall has been removed?

Reply:
postinstall is no longer recommended (it it faster to process libs that application will import than whole node_modules)
https://github.com/angular/angular/blob/master/CHANGELOG.md#910-2020-03-25

Reply:
`ngcc` produced a warning because of a "solution-style" tsconfig. When I tried to resolve the warning, I found the PR https://github.com/angular/angular/pull/38003 introducing it and found [this comment](https://github.com/angular/angular/pull/38003#issuecomment-663015965) of an angular team member:

>Currently our docs do not recommend running ngcc as a postinstall hook, except for a particular scenario related to Angular Universal: https://angular.io/guide/ivy#ivy-and-universalapp-shell.

I tried without the `ngcc` postinstall and it did work without any issues. Therefore I have removed it.

Reply:
Interesting... The `ngcc` here was rather for example than because of actual need. The thing is that Angular CLI builders (including Karma) use Angular tool chain on order to prepare files for tests, while Jest doesn't. This means that there is no way to run `ngcc` programmatically only on libraries that you need (because `ts-jest` doesn't utilize Angular tool chain). 
Well, there is a way but it's very complicated and requires major rewrite of `ts-jest`. 
Hence, if your app uses incompatible libs (that were not compiled with Ivy) you have to run `ngcc` manually on these libs.  
I guess we can remove it from the examples and add to the docs, this should be enough.